x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -229,17 +226,20 @@ export const CaseViewActivity = ({
             onSubmit={onSubmitTags}
             isLoading={isLoading && loadingKey === 'tags'}
           />
-          {pushToServiceAuthorized && userActionsData ? (
+          {pushToServiceAuthorized &&
+          userActionsData &&
+          caseConnectors &&
+          caseData &&
+          allAvailableConnectors ? (
             <EditConnector
               caseData={caseData}
-              caseServices={userActionsData.caseServices}
-              connectorName={connectorName}
-              connectors={connectors}
-              hasDataToPush={userActionsData.hasDataToPush}
```

Comment:
Moved the logic inside the `usePushToService` hook which is used by the `EditConnector` component